docs/secure-connections/sauce-connect-5/faq.md

```diff
@@ -6,147 +6,57 @@ sidebar_label: FAQ
-
-For all platforms, tests may perform better when using a locally defined domain name (which can be set in your [`hosts` file](http://en.wikipedia.org/wiki/Hosts_file)) rather than `localhost`. Using a locally defined domain name also allows access to apps on any port. If you are using a parent proxy with Sauce Connect, you may need the [`--proxy-localhost` flag](/dev/cli/sauce-connect-5/run/#proxy-localhost).
+Please refer to our [localhost proxying guide](/secure-connections/sauce-connect-5/guides/localhost-proxying).
 
 ## How can I improve tunnel performance?
 
-See [Improving Sauce Connect Proxy Performance](/secure-connections/sauce-connect-5/guides/tunnels/#sc5perf).
-
-## What are the supported ports on localhost?
-
-See [Ports for proxying localhost traffic](/secure-connections/sauce-connect-5/guides/localhost-proxying).
-
-## What are the optimal open file settings? {#sc5openfiles}
-
-The [open file limit](https://www.tecmint.com/increase-set-open-file-limits-in-linux/) on macOS and other Unix-based systems limits the number of open file descriptors. Since a file descriptor is opened for each network connection, the open file limit also restricts the number of open network connections. Under normal usage, Sauce Connect Proxy can easily reach the default limit (1024 in most cases). This causes each new connection to time out, and results in failed tests. Modern systems allow much higher limits without any visible impact on memory, so we **recommend setting the open file limit to 64000** in order to ensure the open file limit is never an issue for tunneled requests.
-
-## (CIA) If we have five users, should we use five instances of Sauce Connect Proxy or set up one shared instance?
-
-Feel free to do either, even if you only have one Sauce account.
-
-If you do decide to use five separate instances, you'll need to create unique identifiers for each. More information: **High Availability Sauce Connect 5 Setup needed**.
+To improve Sauce Connect 5 tunnel performance, start by ensuring that the instance running Sauce Connect has sufficient system resources. You can find recommended CPU and memory requirements in [System Requirements](/secure-connections/sauce-connect-5/system-requirements/#cpu-and-memory-resources).
 
-## (CIA) Can I split testing traffic across multiple Sauce Connect Proxy tunnels?
+Another effective way to optimize performance is to reduce unnecessary traffic passing through the tunnel. You can use the [-F, --deny-domains](/dev/cli/sauce-connect-5/run#deny-domains) flag to block domains that do not need to be accessed during testing. This helps limit bandwidth usage and reduces load on the tunnel.
```

Comment:
How about limiting the traffic that goes through the tunnel by using a direct-domains flag? That might significantly improve tunnel performance as well.

Reply:
Good idea. Added.